muschema/src/struct.ts

```diff
@@ -39,23 +39,23 @@ export class MuStruct<StructSpec extends _SchemaDictionary>
         });
 
         const args:string[] = [];
-        const props:any[] = [];
+        const props:MuSchema<any>[] = [];
```

Comment:
injected properties may not be schemas necessarily.  for example, long string constants might be good candidates for injection, or maybe other objects which are initialized in the containing scope.